### PR TITLE
Cart and wishlist 

### PR DESCRIPTION
**Pull request by kushalsourav**
Cart and wishlist added

### CODE REVIEW

**Review by Debarshi95 — APPROVED** [review]
Looks good. Can merge it.

**Review by ShrutiLakhani — APPROVED** [review]
Code looks clean to merge. Pls try and attach either demo link or screenshots for ease of review.

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                        <div class="card-img card-img-1">
+
+                        </div>
+                        <div class="card-abt">
+                            <div class="card-head">
+                                <h3 class="card-title">Men premium jacket</h3>
+                                <h4 class="card-subtitle">₹5000</h4>
+                                <small>50% offer</small>
+                                <div>
+                                    quantity <span><input type="number" class="input-small"></span>
```
Comment (partha8) [inline cart.html]: in the input have value="1" and min="1"

File: wishlist.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+                        </div>
+                        <div class="card-head text-center">
+                            <h3 class="card-title">Vogue</h3>
+                            <span>₹5000</span>
+                        </div>
+                        <div class="card-links">
+                            <button class="btn btn-primary btn-large">Move to cart</button>
+                        </div>
+                        <span class="badge-quinary"><i class="far fa-heart"></i></span>
```
Comment (partha8) [inline wishlist.html]: Here, have the heart icon in red color itself as it's already in the wishlist right?

**Review by partha8 — APPROVED** [review]
looks good, good work! A suggestion would be is that you should start using semantic HTML more often instead of divs.

**Pull request merged**